packages/hash/frontend/src/components/Link.tsx

```diff
@@ -91,7 +92,6 @@ export const Link = React.forwardRef<HTMLAnchorElement, LinkProps>(
       if (noLinkStyle) {
         return (
           <Anchor
-            sx={{ ":hover": { borderBottom: "none" } }}
```

Comment:
Removed this @akash-joshi because this was an override for the hover styles applied by the previous global styles config. We've removed that and as a result no longer need this